### PR TITLE
Czech translation for the tukit

### PR DESCRIPTION
**Pull request by djz88**

### CODE REVIEW

File: po/cz.po [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+"Language: cs\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"#-#-#-#-#  tukit.manifest.pot (PACKAGE_VERSION)  #-#-#-#-#\n"
+"Project-Id-Version: tukit 0.1\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Cockpit manifest2po\n"
```
Comment (skazi0) [inline po/cz.po:31]: Please add `"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2\n"` (see: https://savannah.nongnu.org/bugs/?27430 for some details)

File: po/cz.po [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+#, fuzzy
+msgid ""
+msgstr ""
+"#-#-#-#-#  tukit.html.pot (PACKAGE_VERSION)  #-#-#-#-#\n"
+"Project-Id-Version: tukit 0.1\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Cockpit html2po\n"
+"#-#-#-#-#  tukit.js.pot (PACKAGE_VERSION)  #-#-#-#-#\n"
```
Comment (skazi0) [inline po/cz.po:15]: please simplify the metadata to remove duplicates. check `po/pl.po` for example

Comment (djz88) [reply po/cz.po:15]: I went through template file... maybe it could be simplified as well.

Comment (skazi0) [reply po/cz.po]: The template is generated and updated automatically by the tools.

**Pull request merged**